src/main/java/org/privacyidea/PIResponse.java

```diff
@@ -53,7 +53,8 @@
 
     public boolean pushAvailable()
     {
-        return multichallenge.stream().anyMatch(c -> TOKEN_TYPE_PUSH.equals(c.getType()));
+        return multichallenge.stream()
```

Comment:
these should be on one line since it is only 2 chained calls

Reply:
[54c643e](https://github.com/privacyidea/java-client/pull/54/commits/54c643ea9a5e150b69bc454583b16ae856c99ce0)